crates/air-lib/air-parser/src/ast/values/impls.rs

```diff
@@ -88,7 +88,7 @@ impl<'i> Variable<'i> {
         Self::Stream(Stream::new(name, position))
     }
 
-    pub fn name(&self) -> &str {
+    pub fn name(&self) -> &'i str {
```

Comment:
It's unnecessary, because it's used by methods that didn't required this lifietime declared explicitly, but after validator refactoring it became necessary.